be/src/util/array_parser.h

```diff
@@ -175,6 +202,37 @@ class ArrayParser {
             memory_copy(string_val->ptr, iterator->GetString(), iterator->GetStringLength());
             break;
         }
+        case TYPE_DATE:
+        case TYPE_DATETIME: {
+            DateTimeValue value;
+            value.from_date_str(iterator->GetString(), iterator->GetStringLength());
+            *val = reinterpret_cast<AnyVal*>(context->allocate(sizeof(DateTimeVal)));
+            new (*val) DateTimeVal();
+            value.to_datetime_val(static_cast<DateTimeVal*>(*val));
+            break;
+        }
+        case TYPE_DECIMALV2: {
+            *val = reinterpret_cast<AnyVal*>(context->allocate(sizeof(DecimalV2Val)));
+            new (*val) DecimalV2Val();
+
+            if (iterator->IsNumber()) {
+                if (iterator->IsInt() || iterator->IsUint() || iterator->IsInt64()) {
```

Comment:
does IsInt32 better than IsInt, and IsUint32 better than IsUint?  At the first look, IsInt == IsInt32 || IsInt64.

Reply:
After refinement, it is sufficient to check whether the iterator is `Uint64` here.